src/huggingface_hub/serialization/_torch.py

```diff
@@ -379,7 +379,10 @@ def get_torch_storage_id(tensor: "torch.Tensor") -> Tuple["torch.device", Union[
 
     Taken from https://github.com/huggingface/transformers/blob/1ecf5f7c982d761b4daaa96719d162c324187c64/src/transformers/pytorch_utils.py#L278.
     """
-    return tensor.device, _get_unique_id(tensor), get_torch_storage_size(tensor)
+    if tensor.device.type == "meta":
+        return None
+    else:
+        return tensor.device, _get_unique_id(tensor), get_torch_storage_size(tensor)
```

Comment:
I'm fine with the changes you've made :+1:
Could you just update the docstring just above to reflect this change? Currently it says 


> Multiple different tensors can share the same underlying storage. For example, "meta" tensors all share the same storage, and thus their identifier will all be equal. This identifier is guaranteed to be unique and constant for this tensor's storage during its lifetime. Two tensor storages with non-overlapping lifetimes may have the same id.

which is not true anymore (will always return None)

And thanks for looking into it in the first place!

Reply:
Done !